src/svgs/CreditCardIcon.tsx

```diff
@@ -1,150 +1,16 @@
-
-const DiscoverIcon = props => (
-  <Icon
-    xmlnsXlink="http://www.w3.org/1999/xlink"
-    width={30}
-    height={20}
-    viewBox="0 0 30 20"
-    {...props}
-  >
-    <defs>
-      <rect id="a" width={30} height={20} rx={2} />
-    </defs>
-    <g fill="none" fillRule="evenodd">
-      <use fill="#EFEFEF" xlinkHref="#a" />
-      <rect width={29} height={19} x={0.5} y={0.5} stroke="#E5E5E5" rx={2} />
-      <path
-        fill="#FFB44F"
-        d="M12 20h16.005A1.992 1.992 0 0 0 30 18.002V12l-18 8z"
-      />
-      <rect width={20} height={3} x={5} y={5} fill="#D4D4D4" rx={1.5} />
-    </g>
-  </Icon>
-)
-
-const FallbackIcon = props => (
-  <Icon
-    xmlnsXlink="http://www.w3.org/1999/xlink"
-    width={30}
-    height={20}
-    viewBox="0 0 30 20"
-    {...props}
-  >
-    <defs>
-      <rect id="a" width={30} height={20} rx={2} />
-    </defs>
-    <g fill="none" fillRule="evenodd">
-      <use fill="#EFEFEF" xlinkHref="#a" />
-      <rect width={29} height={19} x={0.5} y={0.5} stroke="#E5E5E5" rx={2} />
-      <rect width={20} height={3} x={5} y={5} fill="#D4D4D4" rx={1.5} />
-      <rect width={7} height={3} x={5} y={10} fill="#D4D4D4" rx={1.5} />
-    </g>
-  </Icon>
-)
-
-const MastercardIcon = props => (
-  <Icon
-    xmlnsXlink="http://www.w3.org/1999/xlink"
-    width={30}
-    height={20}
-    viewBox="0 0 30 20"
-    {...props}
-  >
-    <g fill="none" fillRule="evenodd">
-      <rect
-        width={29}
-        height={19}
-        x={0.5}
-        y={0.5}
-        fill="#EFEFEF"
-        stroke="#E5E5E5"
-        rx={2}
-      />
-      <g transform="translate(6 5)">
-        <circle cx={13} cy={5} r={5} fill="#FFB44F" />
-        <circle cx={5} cy={5} r={5} fill="#CE4747" />
-        <path
-          fill="#D6765F"
-          d="M9 8a4.977 4.977 0 0 1-1-3c0-1.126.372-2.164 1-3 .628.836 1 1.874 1 3a4.977 4.977 0 0 1-1 3z"
-        />
-      </g>
-    </g>
-  </Icon>
-)
-const VisaIcon = props => (
-  <Icon
-    xmlnsXlink="http://www.w3.org/1999/xlink"
-    width={30}
-    height={20}
-    viewBox="0 0 30 20"
-    {...props}
-  >
-    <defs>
-      <rect id="a" width={30} height={20} rx={2} />
-    </defs>
-    <g fill="none" fillRule="evenodd">
-      <use fill="#EFEFEF" xlinkHref="#a" />
-      <rect width={29} height={19} x={0.5} y={0.5} stroke="#E5E5E5" rx={2} />
+/** CreditCardIcon */
+export const CreditCardIcon: React.SFC<IconProps> = props => {
+  return (
+    <Icon {...props} viewBox="0 0 18 18">
+      <title>credit card</title>
       <path
-        fill="#FFB44F"
-        d="M0 14v4.002C0 19.106.898 20 1.992 20h26.016c1.1 0 1.992-.898 1.992-1.998V14H0z"
+        d="M2 6h14V4H2v2zm0 1v7h14V7H2zm-.5-4h15a.5.5 0 0 1 .5.5v11a.5.5 0 0 1-.5.5h-15a.5.5 0 0 1-.5-.5v-11a.5.5 0 0 1 .5-.5zM3 9h4v1H3V9z"
+        fill="#000"
+        fillRule="nonzero"
       />
-      <path
-        fill="#4287CB"
-        d="M1.992 0C.892 0 0 .898 0 1.998V6h30V1.998A1.999 1.999 0 0 0 28.008 0H1.992z"
-      />
-    </g>
-  </Icon>
-)
```

Comment:
Why remove all of this code? This has specific relevance to the Purchase team order flow.

Reply:
I did not notice that there was behavior in here - what I ended up doing was something like this:

```
$ rm -rf src/svgs
$ mv src/new_svgs src/svgs
```

And then just committed all the churn. So what I'm trying to say is that I didn't go file by file and delete things, I did it all in a batch and assumed we'd be good - clearly I was wrong! I'll follow up on some of these and figure out a strategy to not break things.

Reply:
Ahh i see -- I had assumed that once the folder was outputted to `new_icons` things were dragged over manually and then cleaned up by hand. This makes sense now.